requirements.txt

```diff
@@ -7,4 +7,7 @@ pefile == 2023.2.7
 semantic_version == 2.10.0
 GitPython == 3.1.31
 pyyaml == 6.0.0
-cryptography == 40.0.1
\ No newline at end of file
+cryptography == 40.0.1
+openpyxl == 3.1.2
```

Comment:
These need to be added to the setup.py file, install_requires. If this tool only works on Windows, I would go ahead and just add the windows requirement to openpyxl and XlsxWriter too, since they would only ever be used with the tool.